website/users/models.py

```diff
@@ -62,3 +62,15 @@ def __str__(self):
         :return: the username of the user
         """
         return self.username
+
+    def get_short_name(self):
```

Comment:
Doesn't Django come with a built-in alternative for this? Such as display name or something?

Reply:
This is an overwrite for that function to display the first name if present and else fall back to the username

Reply:
Okay sure :)